lib/streamlit/util.py

```diff
@@ -472,6 +472,9 @@ def is_namedtuple(x):
         return False
     return all(type(n).__name__ == "str" for n in f)
 
+def is_pydeck(obj):
+    """True if input looks like a pydeck chart."""
+    return is_type(obj, "pydeck.bindings.deck.Deck")
```

Comment:
These `is_foo` methods have moved to a file called `type_util.py`. Please merge the latest `develop` and move this one there too.